docs/modules.rst

```diff
@@ -176,6 +176,13 @@ sockeye.rnn_attention module
     :members:
     :show-inheritance:
 
+sockeye.score module
+----------------------------
```

Comment:
[nitpick] sphinx will complain that these dashes are longer than the line above (I know its silly)